tests/test.py

```diff
@@ -1043,7 +1043,7 @@ def test_rule_expressions_cuckooreport_context(self):
     def test_rule_expressions_olereport_context(self):
         """ Test generic rule olereport context """
         config = '''[expressions]
-            expression.3  : sample.file_extension in {'doc', 'rtf'} and olereport.has_office_macros == True -> bad
+            expression.3  : sample.file_extension in {'doc', 'rtf', 'rtx'} and olereport.has_office_macros == True -> bad
```

Comment:
Just out of curiosity: What does this addition provide?

Reply:
Addressed in #143 by adding a test for it and adding it to the example rule in `ruleset.conf.sample`.